.github/workflows/build.yml

```diff
@@ -180,7 +180,7 @@ jobs:
         with:
           run: |
             cd sources/testing
-            ./cibuild.sh -x testlist/${{matrix.boards}}.dat
+            ./cibuild.sh testlist/${{matrix.boards}}.dat
```

Comment:
@v01d need modfiy these too:
https://github.com/apache/incubator-nuttx-testing/blob/master/.github/workflows/build.yml#L84
https://github.com/apache/incubator-nuttx-testing/blob/master/.github/workflows/build.yml#L142
How about we merge incubator-nuttx-testing into nuttx/tools which will simplify the furture build enhancement a lot. BTW, it's also very helpful for people who want to setupt the local CI infrastructure.